src/components/PeopleManagement/DeleteGroupModal.jsx

```diff
@@ -0,0 +1,88 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { useParams } from 'react-router-dom';
+import { FormattedMessage } from '@edx/frontend-platform/i18n';
+import { logError } from '@edx/frontend-platform/logging';
+import {
+  ActionRow, Button, ModalDialog, useToggle,
+} from '@openedx/paragon';
+import { RemoveCircleOutline } from '@openedx/paragon/icons';
+
+import GeneralErrorModal from './GeneralErrorModal';
+import { ROUTE_NAMES } from '../EnterpriseApp/data/constants';
+
+import LmsApiService from '../../data/services/LmsApiService';
+
+const DeleteGroupModal = ({
+  group, isOpen, close,
+}) => {
+  const { enterpriseSlug } = useParams();
+  const [isErrorOpen, openError, closeError] = useToggle(false);
+  const removeEnterpriseGroup = async () => {
+    try {
+      await LmsApiService.removeEnterpriseGroup(group?.uuid);
+      close();
+      // redirect back to the people management page
+      window.location.href = `/${enterpriseSlug}/admin/${ROUTE_NAMES.peopleManagement}`;
+    } catch (error) {
+      logError(error);
+      openError();
+    }
+  };
+  return (
+    <>
+      <GeneralErrorModal
+        isOpen={isErrorOpen}
+        close={closeError}
+      />
+      <ModalDialog
+        title="Delete group"
+        isOpen={isOpen}
+        onClose={close}
+        hasCloseButton
+        isFullscreenOnMobile
+      >
+        <ModalDialog.Header>
+          <ModalDialog.Title>Delete group?</ModalDialog.Title>
+        </ModalDialog.Header>
+        <ModalDialog.Body>
+          <p className="pb-3">
+            <FormattedMessage
+              id="people.management.delete.group.modal.body.1"
+              defaultMessage="This action cannot be undone."
+              description="Warning shown when deleting a group."
+            />
+          </p>
+          <FormattedMessage
+            id="people.management.delete.group.modal.body.2"
+            defaultMessage={
+            'By deleting this group you will no longer be able to track analytics associated'
+            + 'with this group and the group will be removed from your People Management page.'
```

Comment:
minor change here to add a space in front of `with` or after `associated` so that it doesn't look like one word.
![Screenshot 2024-10-15 at 8 31 51 AM](https://github.com/user-attachments/assets/5950b8e5-ad0a-4de8-a938-a7b550632763)